internal/infrastructure/elasticsearch/elastic_search_service.go

```diff
@@ -69,9 +69,10 @@ func (d *ElasticSearch) buildAndExecRequest(method, url string, obj interface{},
 	req.Header.Set("Authorization", "ApiKey "+d.Token)
 	var resp *http.Response
 	var err error
```

Comment:
Does this `err` ever get assigned to before line 106? In particular, it seems like `err == nil` on line 76 will always be true, and `err != nil` will always be false.

Reply:
I think the worst outcome here is that we fail a few times due to, say, timeout; we didn't pass in an `outObj`, and we return with a nil error like everything is time.